workspaces/docs/docusaurus.config.js

```diff
@@ -1,57 +1,57 @@
 // @ts-check
 // Note: type annotations allow type checking and IDEs autocompletion
 
-const lightCodeTheme = require('prism-react-renderer/themes/github');
-const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+const lightCodeTheme = require("prism-react-renderer/themes/github");
+const darkCodeTheme = require("prism-react-renderer/themes/dracula");
 
 /** @type {import('@docusaurus/types').Config} */
 const config = {
-  title: 'calendar-widgets',
+  title: "calendar-widgets",
   tagline:
-    'A collection of calendar widgets and utilities to make working with calendars less shitty',
-  favicon: 'img/favicon.ico',
-  url: 'https://calendar-widgets.com',
-  baseUrl: '/',
+    "A collection of calendar widgets and utilities to make working with calendars less shitty",
+  favicon: "",
```

Comment:
it looks like this being an empty string is causing the Docusaurus build to fail. Can we remove this field entirely? https://docusaurus.io/docs/api/docusaurus-config#favicon

Reply:
Hmm, I see.
Will check that.